client/components/Account/AddCredentialForm.vue

```diff
@@ -12,13 +12,17 @@ export default {
       method: 'POST',
       setAccount: true,
       fields: [
-        {id: 'username', label: 'Username'},
-        {id: 'password', label: 'Password', type: 'password'}
+        {id: 'username', label: 'New Username'},
+        {id: 'password', label: 'New Password', type: 'password'}
       ],
-      title: 'Create a user and add to the shared account',
+      validators: {
+        username: this.$helpers.validators.username,
+        password: this.$helpers.validators.password,
+      },
+      title: 'Add new person to the shared account',
       callback: () => {
         this.$store.commit('alert', {
-          message: 'Successfully created the new user and added to your account!',
+          message: 'Successfully added the new user to your shared account!',
```

Comment:
I think having "created and added" might actually be more helpful for our users

Reply:
ok I changed this to "person" to avoid any ambiguity. We now don't mention "user" anywhere in the frontend code.